RNA-seq/05-nb_cell_line_DESeq2.Rmd

```diff
@@ -212,10 +176,7 @@ deseq_df %>%
   dplyr::arrange(dplyr::desc(stat))
 ```
 
-Let's take a look at what the two top genes are:
-
-* [`ENSG00000233718`](https://useast.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000233718;r=2:15921037-15942249)
-* [`ENSG00000134323`](http://useast.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000134323;r=2:15940564-15947007)
+Let's take a look at what the top gene is: [`ENSG00000134323`](http://useast.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000134323;r=2:15940564-15947007)
```

Comment:
I like to minimize these links; the last part is just the zoom region for the map, and it will pick a good one automatically (even as coordinates change). 
```suggestion
Let's take a look at what the top gene is: [`ENSG00000134323`](http://useast.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000134323)
```